cmd/call/call.go

```diff
@@ -39,9 +39,11 @@ var CallCmd = &cobra.Command{
 
 func init() {
 	flags := CallCmd.Flags()
-	// Add any flags that were registered on the built-in flag package.
-	flags.AddGoFlagSet(flag.CommandLine)
```

Comment:
I've removed this because it adds unwanted flags that are set by libraries we use (Ex: glog). I consider that if we want to let the user use those flags, we should explicitly let them. 
Besides, those flags cant be imported to projects importing gurl.